packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -28,6 +28,7 @@ Future<void> buildWeb(
   String serviceWorkerStrategy,
   bool sourceMaps,
   bool nativeNullAssertions,
+  String baseHref
```

Comment:
nit: trailing comma:
```suggestion
  String baseHref,
```